ballerina/websocket_service.bal

```diff
@@ -58,7 +58,7 @@ isolated service class WsService {
         }
 
         WSPayload|json|error wsPayload = self.customHeaders != {}
-                                        ? wsText.cloneWithType(WSPayload) : value:fromJsonString(text);
+            ? wsText.cloneWithType(WSPayload) : value:fromJsonString(text);
```

Comment:
Any reason to change the indentation here?

Reply:
Since this file has lot of changes I have formatted it with ballerina. The previous one is not properly indented